### PR TITLE
fix predict function

### PR DESCRIPTION
**Pull request by AndreiXYZ**
Fixed predict function to return proper labels. Done so replicating the code [from here](https://huggingface.co/Wellcome/WellcomeBertMesh)

### CODE REVIEW

**Comment by AndreiXYZ** [comment]
To reproduce:
```bash
grants_tagger predict \
"Plant-based repellents have been used for generations in traditional practice as a personal protection measure against host-seeking mosquitoes. Knowledge on traditional repellent plants obtained through ethnobotanical studies is a valuable resource for the development of new natural products. Recently, commercial repellent products containing plant-based ingredients have gained increasing popularity among consumers, as these are commonly perceived as “safe” in comparison to long-established synthetic repellents although this is sometimes a misconception. To date insufficient studies have followed standard WHO Pesticide Evaluation Scheme guidelines for repellent testing. There is a need for further standardized studies in order to better evaluate repellent compounds and develop new products that offer high repellency as well as good consumer safety. This paper presents a summary of recent information on testing, efficacy and safety of plant-based repellents as well as promising new developments in the field." \
"Wellcome/WellcomeBertMesh" \
--threshold 0.5
```

**Review by nsorros — CHANGES_REQUESTED** [review]
- Uncommment tests and make them pass
- Add test for the erratic behaviour you were experiencing

File: tests/test_cli.py [hunk 1]
```diff
@@ -218,24 +212,21 @@ def test_pretrain_command():
...
-        save_transformer_model_to_path(model_path)
+    model_path = "Wellcome/WellcomeBertMesh"
 
-        text = "malaria"
-        result = runner.invoke(app, ["predict", text, model_path])
-        print(result)
-        assert result.exit_code == 0
+    text = "malaria"
+    result = runner.invoke(app, ["predict", text, model_path])
+    print(result)
```
Comment (nsorros) [inline tests/test_cli.py:219]: remove

File: tests/test_predict.py [hunk 2]
```diff
@@ -1,35 +1,10 @@
-# from unittest.mock import patch
-# import tempfile
-# import shutil
-# import pickle
-# import json
-# import os
```
Comment (nsorros) [inline tests/test_predict.py:6]: remove commented code

**Review by nsorros — APPROVED** [review]
Apoproving to unblock but remove print and comments

**Pull request merged**